pkg/redfish/rfcomponents.go

```diff
@@ -1270,16 +1284,21 @@ func (s *EpSystem) discoverRemotePhase1() {
 			}
 		}
 		if nodeChassis.ChassisRF.Power.Oid != "" {
+			FoxconnPowerRetryNum := 1
+			FoxconnPowerRetryDelay := 10
+
+			FoxconnPowerRetry:
```

Comment:
I think the vars and the label should go above the if. A goto shouldn't break into a block of code bypassing it's prepositions.

Reply:
In this case, the vars and this label (FoxconnPowerRetry) are local, within the if block scope.  The goto is staying within that scope.  If we were to place them above the if statement then we'd be breaking outside that scope even though we want to stay within it.  Thoughts?

Reply:
You're right. I didn't notice that the goto was also in this if block.